### PR TITLE
[CoreLocation] Added support for Xcode 14.1 b1

### PR DESCRIPTION
**Pull request by SotoiGhost**

### CODE REVIEW

File: src/CoreLocation/CLEnums.cs [hunk 1]
```diff
@@ -67,6 +67,10 @@ public enum CLError : long {
 
 		// ios14
 		PromptDeclined = 18,
+
+		// ios16
+		[Mac (13,0), iOS (16,0), MacCatalyst (16,0), Watch (9,0), TV (16,0)]
```
Comment (rolfbjarne) [inline src/CoreLocation/CLEnums.cs:72]: Shouldn't this be:

```suggestion
		[Mac (13,0), iOS (16,1), MacCatalyst (16,0), Watch (9,1), TV (16,1)]
```

File: src/corelocation.cs [hunk 2]
```diff
@@ -474,6 +476,9 @@ partial interface CLLocationManager {
 		[Export ("stopMonitoringLocationPushes")]
 		void StopMonitoringLocationPushes ();
 
+		[Watch (9,0), NoTV, NoMac, NoiOS, NoMacCatalyst]
```
Comment (rolfbjarne) [inline src/corelocation.cs:479]:
```suggestion
		[Watch (9,1), NoTV, NoMac, NoiOS, NoMacCatalyst]
```

**Review by mandel-macaque — APPROVED** [review]

File: src/CoreLocation/CLEnums.cs [hunk 3]
```diff
@@ -67,6 +67,10 @@ public enum CLError : long {
 
 		// ios14
 		PromptDeclined = 18,
+
+		// ios16
+		[NoMac, iOS (16,1), MacCatalyst (16,0), Watch (9,1), TV (16,1)]
```
Comment (tj-devel709) [inline src/CoreLocation/CLEnums.cs:72]: Is it MacCatalyst (16,1) instead of (16,0)?

**Review by tj-devel709 — COMMENTED** [review]
Just a small question about version!

File: src/CoreLocation/CLEnums.cs (hunk 3 above)
Comment (mandel-macaque) [reply src/CoreLocation/CLEnums.cs:72]:
```suggestion
		[NoMac, iOS (16,1), MacCatalyst (16,1), Watch (9,1), TV (16,1)]
```

**Pull request merged**